Penteract/Assets/Scripts/PlayerController.h

```diff
@@ -100,8 +100,9 @@ class PlayerController : public Script
 	float switchDelay = 0.37f;
 
 	/* Fang & onimaru damage */
-	int onimaruDamage = 1;
-	int fangDamage = 3;
+	
+	int rangedDamageTaken = 1;
```

Comment:
Why do we need two variables for damage taken?

Reply:
I think it is to not have hardcoded how much life the player loses depending on which element collides with it. In player death script, on the On Collision function, if it has collided with a bullet instead of taking x damage hardcoded, you take rangedDamageTaken